packages/vscode-extension/src/handlers.ts

```diff
@@ -901,61 +901,94 @@ export async function grantPermission(env: string): Promise<Result<Void, FxError
   return result;
 }
 
-export async function listCollaborator(env: string): Promise<TreeItem[]> {
-  let result: TreeItem[] = [];
+export async function listAllCollaborators(envs: string[]): Promise<Record<string, TreeItem[]>> {
+  const result: Record<string, TreeItem[]> = {};
   ExtTelemetry.sendTelemetryEvent(TelemetryEvent.ListCollaboratorStart);
 
-  try {
-    const checkCoreRes = checkCoreNotEmpty();
-    if (checkCoreRes.isErr()) {
-      throw checkCoreRes.error;
-    }
+  let fatalError;
+  const checkCoreRes = checkCoreNotEmpty();
+  if (checkCoreRes.isErr()) {
+    fatalError = checkCoreRes.error;
```

Comment:
Need to throw the error here? Otherwise, below line 915 will fail as core is undefined/null.

Reply:
Oh, you're right, thanks and will update

Reply:
Updated